src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY_KEYWORD = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToFile(calculateReadDataFromFile(fromFileName), toFileName);
+    }
+
+    private String calculateReadDataFromFile(String fromFileName) {
```

Comment:
This method has an unclear purpose, let's divide it into two separate: first one will read data from file and second one will process that data